inc/admin.php

```diff
@@ -10,6 +10,10 @@
 namespace Authorship\Admin;
 
 use WP_Post;
+use WP_Error;
+use stdClass;
```

Comment:
Need to import WP_User if using it as a typehint for function params as per my later suggestion. 

```suggestion
use WP_Error;
use WP_User;
use stdClass;
```